frontend/src/components/elements/DocString/styled-components.ts

```diff
@@ -16,35 +16,87 @@
 
 import styled from "@emotion/styled"
 
-export const StyledDocModule = styled.span(({ theme }) => ({
-  color: theme.colors.docStringModuleText,
+export const StyledDocSummary = styled.span(({ theme }) => ({
+  "& > *": {
+    marginRight: theme.spacing.sm,
+  },
 }))
 
 export const StyledDocName = styled.span(({ theme }) => ({
   fontWeight: theme.fontWeights.bold,
 }))
 
+export const StyledDocType = styled.span(({ theme }) => ({
+  color: theme.colors.green70,
+}))
+
+export const StyledDocValue = styled.span()
+
 export interface StyledDocContainerProps {
   width: number
 }
 
 export const StyledDocContainer = styled.span<StyledDocContainerProps>(
   ({ theme, width }) => ({
-    backgroundColor: theme.colors.docStringContainerBackground,
-    padding: `${theme.spacing.sm} ${theme.spacing.lg}`,
+    display: "flex",
+    flexDirection: "column",
+    borderRadius: theme.radii.md,
+    border: `1px solid ${theme.colors.fadedText05}`,
     fontFamily: theme.fonts.monospace,
     fontSize: theme.fontSizes.sm,
-    overflowX: "auto",
-    width,
   })
 )
 
 export const StyledDocHeader = styled.div(({ theme }) => ({
-  paddingBottom: theme.spacing.sm,
-  marginBottom: theme.spacing.sm,
-  borderBottom: `1px solid ${theme.colors.fadedText10}`,
+  padding: `${theme.spacing.sm} ${theme.spacing.lg}`,
+  backgroundColor: theme.colors.docStringContainerBackground,
+  fontSize: theme.fontSizes.sm,
+
+  overflow: ["auto", "overlay"],
+
+  "&:not(:last-child)": {
+    borderBottom: `1px solid ${theme.colors.fadedText05}`,
+  },
```

Comment:
Just curious why bottom border is implemented with the not the last child selector. Probably missing something but playing with this in inspect and can't discern the difference between using this selector and directly specifying the bottom border.

Reply:
Oh, I think this is left over from a previous version of the st.help spec I was working with. I'll remove it.

In a previous version of the spec, it was possible for this header element to be the last child of the `DocString` element, and this selector avoided a double bottom border. (This would happen when the object passed to `st.help( <-- here --> )` doesn't have a docstring or member variables/methods.)

However, since we now show a placeholder saying "No docs available" when there's no docstring, this header can never be the last child anymore.